addons/web/views/report_templates.xml

```diff
@@ -384,17 +384,9 @@
                     <img t-if="company.logo" t-att-src="image_data_uri(company.logo)" alt="Logo"/>
                 </div>
                 <div class="col-5 offset-1" name="company_address">
-                    <ul class="list-unstyled">
-                        <strong><li t-if="company.name"><span t-field="company.name"/></li></strong>
-                        <li t-if="forced_vat or company.vat">
-                            <t t-esc="company.country_id.vat_label or 'Tax ID'"/>:
-                            <span t-if="forced_vat" t-esc="forced_vat"/>
-                            <span t-else="" t-field="company.vat"/>
-                        </li>
-                        <li t-if="company.phone">Tel: <span class="o_force_ltr" t-field="company.phone"/></li>
-                        <li t-if="company.email"><span t-field="company.email"/></li>
-                        <li t-if="company.website"><span t-field="company.website"/></li>
-                    </ul>
+                    <div name="company_address">
+                        <span t-if="company.company_details" t-field="company.company_details"></span>
+                    </div>
```

Comment:
Don't put a div with the name company_address alone inside a div named company_address too :smile: 
I think you can just remove the inner div completely

Reply:
Oops, indeed